Makefile

```diff
@@ -1,17 +1,28 @@
+SHELL=/bin/bash
+
 CFLAGS= -Wall -Wextra
 CFLAGS+= -O0 -ggdb
 CFLAGS+= -std=c99
 
 LDFLAGS= -lxcb-randr -lxcb
 
-all: xrandr-invert-colors.bin
+PREFIX=/usr/local/bin
```

Comment:
I propose here a question mark before the equal sign, so that `PREFIX=/something ; make install ` will install the program into /something: `PREFIX?=/usr/local/bin`